src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/TraceEventExtensions.cs

```diff
@@ -344,6 +413,11 @@ private static void HandleHistogram(TraceEvent obj, CounterConfiguration configu
             string unit = (string)obj.PayloadValue(4);
             string tags = (string)obj.PayloadValue(5);
             string quantilesText = (string)obj.PayloadValue(6);
+            int? id = null;
+            if (obj.Version >= 2)
+            {
+                id = (int)obj.PayloadValue(9);
```

Comment:
nit: might be worth a comment that this is intentionally 9 since at a glance it seems odd to skip over 7 and 8 (before I checked `MetricsEventSource` I thought it could have been a copy-paste error from a different type of counter)